doc/examples.xml

```diff
@@ -309,3 +309,45 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="BananaTreeGraph">
+<ManSection>
+  <Oper Name="BananaTreeGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph</Returns>
+  <Description>
+    If <A>n</A> and <A>n</A> are positive integers,
+    then this operation returns the <E>banana tree graph</E> 
+    <M>BananaTreeGraph(<A>m</A>, <A>n</A>)</M>. <P/>
+
+    From
+    <URL>https://mathworld.wolfram.com/BananaTree.html</URL>:
+    <P/>
+
+    <Q>A banana tree graph, as defined by Chen et al. (1997) is a graph obtained by
+    connecting one leaf of each of <A>m</A> copies of an <A>n</A>-star graph with
+    a single root vertex that is distance from all the stars.</Q>
+    <P/>
+
+    If the optional first argument <A>filt</A> is present, then this should
+    specify the category or representation the digraph being created will
+    belong to. For example, if <A>filt</A> is <Ref Filt="IsMutableDigraph"/>,
+    then the digraph being created will be mutable, if <A>filt</A> is <Ref
+    Filt="IsImmutableDigraph"/>, then the digraph will be immutable. 
+    If the optional first argument <A>filt</A> is not present, then <Ref
+      Filt="IsImmutableDigraph"/> is used by default.<P/>
+
+    See also <Ref Oper="BananaTreeGraph"/>.
```

Comment:
Is this meant to refer to something else? Because this is already the documentation for `BananaTreeGraph`. (Maybe it should be `StarGraph`, for when #424 is merged?)